### PR TITLE
chore: pin ember-truth-helpers

### PR DESCRIPTION
**Pull request by Yelinz**
Pin truth helpers to fix the pipeline

### CODE REVIEW

**Comment by anehx** [comment]
What exactly is the issue?

**Comment by Yelinz** [comment]
Ember truth helpers were updated to 3.0.0, some packages updated but ember-validated-form still has the older version

**Comment by anehx** [comment]
If it doesn't create any errors then configure dependency lint: https://github.com/salsify/ember-cli-dependency-lint#allowed-versions

**Comment by Yelinz** [comment]
Just configuring dependency lint doesnt seem to work

**Comment by Yelinz** [comment]
The pinning can be removed after ember-validated-forms has updated their version of ember-truth-helpers

**Comment by anehx** [comment]
> The pinning can be removed after ember-validated-forms has updated their version of ember-truth-helpers

You're aware that we are the maintainers of that addon, right? :wink:

**Comment by Yelinz** [comment]
Yes but that addon needs some maintenance done

**Comment by anehx** [comment]
Ah come on, that update will cost you about 10 minutes.. I just checked the changelog of `ember-truth-helpers` there are no breaking changes other than node support drops and an update to babel 7, which is already in use.

**Comment by Yelinz** [comment]
Updated it but now the problem lies with ember-cli-addon-docs which doesnt have a newer version

File: package.json [hunk 1]
```diff
@@ -65,20 +65,20 @@
...
-    "@adfinis-sygroup/semantic-release-config": "2.1.4",
+    "@adfinis-sygroup/semantic-release-config": "3.0.1",
     "@babel/plugin-proposal-object-rest-spread": "7.12.1",
     "@ember/jquery": "1.1.0",
     "@ember/optional-features": "2.0.0",
     "@glimmer/component": "1.0.2",
     "babel-eslint": "10.1.0",
     "broccoli-asset-rev": "3.0.0",
-    "ember-cli": "3.22.0",
+    "ember-cli": "3.21.2",
```
Comment (anehx) [inline package.json:75]: Why downgrade?

Comment (Yelinz) [reply package.json:75]: So its equal with ember-source and the ember cli update command works for the next update

Comment (anehx) [reply package.json:75]: Okay, next time open a separate PR for this. This doesn't have anything to do with pinning ember-truth-helpers

**Pull request merged**